lib/elixir/lib/code.ex

```diff
@@ -3,11 +3,19 @@ defmodule Code do
   Utilities for managing code compilation, code evaluation and code loading.
 
   This module complements [Erlang's code module](http://www.erlang.org/doc/man/code.html)
-  to add behaviour which is specific to Elixir.
+  to add behaviour which is specific to Elixir. Almost all of the functions in this module
+  have global side effects on the behaviour of Elixir. 
   """
 
   @doc """
   Lists all loaded files.
+
+  ## Examples
+
+      > Code.require_file("../eex/test/eex_test.exs")
```

Comment:
When there are no doctests, the convention in the codebase is to write the code regularly and use `#=>` for output value:

```elixir
Code.require_file("../eex/test/eex_test.exs")
Code.loaded_files |> List.last |> Kernel.is_bitstring #=> true
```

Although for this particular case, I would rather write something like this:

```elixir
Code.require_file("../eex/test/eex_test.exs")
List.first(Code.loaded_files) =~ "eex_test.exs"
```